templates/default

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
I just asked someone in the office about this and he said "this is very dangerous since a lot of people select their config based off the `NODE_ENV` variable." I know I have pushed some breaking changes, but those are usually CLI flag compatibility changes. I think what I'll do keep this file but leave it blank so that other tools can add and remove lines as they please.